dev-python/qpageview/qpageview-0.6.2.recipe

```diff
@@ -0,0 +1,65 @@
+SUMMARY="Widget to display page-based documents for Qt5/PyQt5"
+DESCRIPTION="qpageview provides a page based document viewer widget for Qt5/PyQt5.
+It has a flexible architecture potentionally supporting many formats. Currently, it supports SVG \
+documents, images, and, using the Poppler-Qt5 binding, PDF documents."
+HOMEPAGE="https://qpageview.org/"
+COPYRIGHT="2009-2022 Wilbert Berendsen"
+LICENSE="GNU GPL v3"
+REVISION="1"
+SOURCE_URI="https://pypi.python.org/packages/source/q/qpageview/qpageview-$portVersion.tar.gz"
+CHECKSUM_SHA256="1c1e3d6dc5374264e1f7c7d4a78616a811e8131c7ed214aed7e08a71f12923e9"
+SOURCE_DIR="qpageview-$portVersion"
+PYTHON3_VERSION="3.8"
+
+ARCHITECTURES="any"
+
+PROVIDES="
+	qpageview = $portVersion
+	"
+REQUIRES="
+	haiku
+	"
+PROVIDES_python38="
+	qpageview_python38 = $portVersion
+	"
+REQUIRES_python38="
+	haiku
```

Comment:
Seems like `haiku` would be duplicated here because it also includes `$REQUIRES` which has `haiku` in it.  Not a big deal.